src/vs/workbench/services/editor/browser/editorService.ts

```diff
@@ -991,27 +984,23 @@ export class EditorService extends Disposable implements EditorServiceImpl {
 				override = replacement.replacement.options?.override;
 			}
 
-			// Resolve the override if not disabled
+			// Resolve override unless disabled
 			if (override !== EditorOverride.DISABLED) {
-				const [overrideResult, , resolvedOptions] = await this.doResolveEditor(
+				const [resolvedEditor] = await this.doResolveEditor(
```

Comment:
Here I wonder if the `EditorGroup` can change? We never seem to pick it up from the resolve call, but what if a picker opens, does that possibly result in a different group to take?

Reply:
We abandoned the group changing by clicking the picker differently to avoid complicating things. That was the only case.